engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -300,4 +295,15 @@ public int compare(RenderableChunk o1, RenderableChunk o2) {
         }
     }
 
+	@Override
+	public RenderingConfig getRenderingConfig() {
```

Comment:
Scrap all of the above. Having discussed these matters over the past 90 minutes or so, I now believe you should pass the WorldProvider and the RenderingConfig instances to the camera's constructor. 

No point in using injection in this circumstance given that the WorldRenderer instance already has references to them.

Sorry about the flip-flopping.